interop-tests/src/test/scala/io/grpc/testing/integration/test/SSLContextUtils.scala

```diff
@@ -0,0 +1,20 @@
+package io.grpc.testing.integration.test
+
+import java.io.{ IOException, InputStream }
+
+import io.grpc.netty.shaded.io.grpc.netty.GrpcSslContexts
+import io.grpc.netty.shaded.io.netty.handler.ssl.{ JdkSslContext, SslProvider }
+import javax.net.ssl.SSLContext
+
+object SSLContextUtils {
+
+  def sslContextForCert(certPath: String): SSLContext = {
+    // Use Netty's SslContextBuilder internally to help us construct a SSLContext
+    val fullCertPath = "/certs/" + certPath
+    val certStream: InputStream = getClass.getResourceAsStream(fullCertPath)
```

Comment:
I think requiring users to add their CA's pem on `/certs/` is a bit limiting.

Could we refactor this to 

```
def sslContextForCert(trustCertResourceName: String) ={
  val certStream: InputStream = getClass.getResourceAsStream(trustCertResourceName)
  ...
}
```
?

Or am I missing something and the limit is on purpose?

Reply:
And also allow for certs with a path not on the classpath would be good, when we make it not a test-utility :)